### DISCUSSION TITLE
Can i replace vvveb editor with vvebjs editor?

### DISCUSSION THREAD

**Discussion by telmob** [givanz/Vvveb#174] [Q&A] [answered]
I've been testing vvveb and i've been unable to change images. So i tested vvvebjs editor and it works without a glitch.
Can i replace the editor in vvveb?

**Comment by givanz** [comment, marked as answer]
Thanks for the bug report, it's fixed in the last update https://github.com/givanz/Vvveb/commit/576a3124f90290dcb2968585dd48245466884fc3 

VvvebJs is embeded in Vvveb CMS with very few code additions for features like revisions etc, the code is mostly the same.
